docs/source/guide/pec-3-options.myst

```diff
@@ -0,0 +1,414 @@
+In the previous section we defined the representation of the Hadamard gate.
+So, for simplicity, we consider a circuit that contains only Hadamard gates.
+
++++
+
+### Defining a circuit of interest and an Executor
+
+```{code-cell} ipython3
+circuit = qiskit.QuantumCircuit(1)
+for _ in range(4):
+    circuit.h(0)
+print(circuit)
+```
+
+In this case, the list of {class}`.OperationRepresentation`s that we need for PEC is simply:
+
+```{code-cell} ipython3
+representations = [h_rep]
+```
+In general `representations` will contain as many representations as the number of ideal
+gates involved in `circuit`.
+
+**Note:** *If a gate is in `circuit` but its {class}`.OperationRepresentation` is not listed in 
+`representations`, Mitiq can still apply PEC. However, any errors associated to
+that gate will not be mitigated. In practice, all the gates without {class}`.OperationRepresentation`s 
+are treated by Mitiq as if they were noiseless.*
+
+The executor must be defined by the user since it depends on the specific frontend and backend
+(see the [Executors](guide-executors.myst) section).
+Here, for simplicity, we import the basic {func}`.execute_with_noise` function from the Qiskit utilities of Mitiq.
+
+```{code-cell} ipython3
+from mitiq import Executor
+from mitiq.interface.mitiq_qiskit import execute_with_noise, initialized_depolarizing_noise
+
+def execute(circuit):
+    """Returns Tr[ρ |0⟩⟨0|] where ρ is the state prepared by the circuit
+    executed with depolarizing noise.
+    """
+    circuit_copy = circuit.copy()    
+    noise_model = initialized_depolarizing_noise(BASE_NOISE)
+    projector_on_zero = [[1, 0], [0, 0]]
+    return execute_with_noise(circuit_copy, projector_on_zero, noise_model)
+
+# Cast the execute function into an Executor class to record the execution histroy
+executor = Executor(execute)
+```
+
+### Options for estimating expectation values
+
++++
+
+In the ["How do I use PEC?"](pec-1-intro.myst) section, we have shown how to apply PEC
+with the minimum amount of options by simply calling the function {func}`.execute_with_pec()`
+with the basic arguments `circuit`, `executor`, and `representations`.
+
+In the next code-cell, we show additional options that can be used:
+
+```{code-cell} ipython3
+pec_value, pec_data = pec.execute_with_pec(
+    circuit,
+    executor,
+    observable=None, # In this example the observable is implicit in the executor
+    representations=representations,
+    num_samples=5, # Number of PEC samples
+    random_state=0, # Seed for reproducibility of PEC sampling
+    full_output=True, # Return "pec_data" in addition to "pec_value"
+)
+```
+
+Similar to other error mitigation modules, `observable` is an optional argument of 
+{func}`.execute_with_pec`. If `observable` is `None` the executor must return an expectation value,
+while if `observable` is a Mitiq {class}`.Observable` the executor must return raw bitstrings. 
```

Comment:
"otherwise the executor must return a `mitiq.QuantumResult` from which the expectation value can be computed with the observable."